source/reference/minio-server/minio-server.rst

```diff
@@ -192,6 +192,11 @@ The command accepts the following arguments:
         - Path to the user's private key file.
         - Absolute path or relative path from current location to the key file to use.
  
+      * - ``trusted-user-ca-key``
+        - Specifies a file containing public key of a certificate authority that is trusted to sign user certificates for authentication.
+          The file must contain a user principals list, and the list must include the user that is authenticating. 
```

Comment:
I'm not sure what "user principals" means here, is that something a person who knows more about certs would be familiar with?

Adding definitions in a table cell isn't the best, however.

Reply:
Added a link